xarray/tests/test_backends.py

```diff
@@ -5496,24 +5496,26 @@ def test_encode_zarr_attr_value() -> None:
 
 @requires_zarr
 def test_extract_zarr_variable_encoding() -> None:
+    # The region is not useful in these cases, but I still think that it must be mandatory
```

Comment:
If the `region` kwarg is required now, should we make it mandatory? Or at least add a TODO in the function if we want to push that off to another PR?

Reply:
I agree, probably, it would be good to add a TODO, indicating that the validation should be outside the extraction of the encoding and that would avoid the use of the region inside the extract_zarr_variable_encoding function

Reply:
If needed I can add a TODO indicating that it would be good to apply that modification.